zipkin/src/main/scala/org/http4s/zipkin/algebras/CollectorOp.scala

```diff
@@ -0,0 +1,14 @@
+package org.http4s.zipkin.algebras
+
+import org.http4s.zipkin.models.ZipkinInfo
+
+import scalaz.Free
+
+// We might consider just making this a trait with an abstract `send` method.
+sealed trait CollectorOp[A]
+
+object CollectorOp {
+  final case class Send(zipkinInfo: ZipkinInfo) extends CollectorOp[Unit]
+  def send(zipkinInfo: ZipkinInfo): Collector[Unit] =
+    Free.liftFC(Send(zipkinInfo))
```

Comment:
Some things are not source compatible.  We've been fortunate that all parts of scalaz that we use thus far are.  Travis should be able to tell us...

... and the answer is [no](https://travis-ci.org/http4s/http4s/jobs/156070055#L717).  We could do a scalaz-7.1 and scalaz-7.2 folder for this, and any other code, that is incompatible.  specs2 has [an example](https://github.com/etorreborre/specs2/tree/master/matcher-extra/src/main) of doing this, which is more complicated than anything we'd need.